LunrCoreLmdb/KeyBuilder.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Text;
+
+namespace LunrCoreLmdb
+{
+    internal static class KeyBuilder
+    {
+        #region Fields
+
+        private const string FieldPrefix = "F:";
+
+        public static byte[] BuildAllFieldsKey() => Encoding.UTF8.GetBytes(FieldPrefix);
+
+        public static byte[] BuildFieldKey(string field) => Encoding.UTF8.GetBytes($"{FieldPrefix}{field}");
```

Comment:
The interpolation is wasteful, just concat the two strings. Might be even better to concat byte arrays, since the left part is constant.

Same applies to all others below.

Reply:
Fixed.